scaleway/resource_lb_beta.go

```diff
@@ -36,10 +37,17 @@ func resourceScalewayLbBeta() *schema.Resource {
 				Optional:    true,
 				Description: "Array of tags to associate with the load-balancer",
 			},
+			"keep_ip_on_delete": {
```

Comment:
I suggest we rename it to `release_ip_on_deletion` because the default value should be false (even if it's a computed field). In the v2, this attribute disappear and an IP will never be released other than when deleting an IP resource.